spec/graphql/mutations/rock_and_pebble/decline_rock_pebble_spec.rb

```diff
@@ -7,6 +7,7 @@
       @user_2 = create :user
       @user_3 = create :user
       @user_1.pebbles << [@user_2, @user_3]
+      RockAndPebble.last.update(active: true)
```

Comment:
Question about this test, not this specific line; would it possible to add an assertion confirming that the email has been sent (or at least added to the pipeline/queue or whatever it's called)? I vaguely recall being able to do that on one of our mod 3 projects but can't remember which.

Reply:
(We could also use stubs to verify that the message gets where it's supposed to if we wanted to isolate this test from that whole process)

Reply:
While I think this is a good idea I have not implemented it yet. We don't do this for any other mutations that send out an email so perhaps this should be it's own ticket? To update all necessary tests?

Reply:
Yeah, that makes sense to me too. I'll make an issue for it